browser/resources/settings/shortcuts_page/commands.tsx

```diff
@@ -108,7 +98,7 @@ function App() {
           <Command key={c.id} command={c} />
         ))}
       </CommandsContainer>
-      <Button kind="plain-faint" onClick={() => commandsCache.resetAll()}>
+      <Button kind="plain-faint" onClick={() => commandsApi.actions.commands.resetAccelerators()}>
```

Comment:
Do I need to go this deep for the actions? Or could I just define the actions directly on the actions object rather than in a `commands` sub property